gci/gitorg.py

```diff
@@ -85,3 +87,14 @@ def get_issue(url):
         print('Unable to load issue %s' % url)
         return
     return issue
+
+
+def get_logo(org_name, size=0):
+    if size != 0:
+        image_url_short = 'http://github.com/%s.png?size=%s' % (org_name, size)
+    else:
+        image_url_short = 'http://github.com/%s.png' % (org_name)
+
+    # Follow the redirect to the page containing the image and
```

Comment:
this comment isnt particularly useful.

Reply:
Sorry - that was from when I was using urllib